manim/mobject/svg/tex_mobject.py

```diff
@@ -253,12 +253,11 @@ def __init__(
             self.tex_to_color_map = {}
         self.tex_environment = tex_environment
         self.brace_notation_split_occurred = False
-        tex_strings = self.break_up_tex_strings(tex_strings)
-        self.tex_strings = tex_strings
+        self.tex_strings = self.break_up_tex_strings(tex_strings)
         try:
             SingleStringMathTex.__init__(
                 self,
-                self.arg_separator.join(tex_strings),
+                self.arg_separator.join(self.tex_strings),
```

Comment:
Is this change from `tex_string` to `self.tex_string` really necessary? It doesn't seem to be accessed anywhere else.

Reply:
Not at all, just removing unnecessary code. I can revert this if you like.

Reply:
Wait, I've read it wrong – I thought you've changed it with this PR so that `tex_strings` is an attribute now, forget my question; it's cleaner this way.